### PR TITLE
Added side navbar

### PR DESCRIPTION
**Pull request by nsk515**
## Reviewers:
@roshnee7

### CODE REVIEW

File: src/Components/Charts.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                <div className='column column-page'>
+                    <h1 style={{paddingLeft:'30px'}}>This is Charts Page</h1>
+                    {msg}
+                </div>
+            </div>
+        )
+    }
+}
+
+export default Charts;
```
Comment (roshnee7) [inline src/Components/Charts.js]: fix end of file

File: src/Components/Configure.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                <div className='column column-page'>
+                    <h1 style={{paddingLeft:'30px'}}>This is Config Page</h1>
+                    {msg}
+                </div>
+            </div>
+        )
+    }
+}
+
+export default Configure;
```
Comment (roshnee7) [inline src/Components/Configure.js]: fix end of file

File: src/Components/Home.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+                <div className='column column-page'>
+                    <h1 style={{paddingLeft:'50px'}}>This is Home Page</h1>
+                    {msg}
+                </div>
+            </div>
+        )
+    }
+}
+
+export default Home;
```
Comment (roshnee7) [inline src/Components/Home.js]: fix end of file

File: src/Components/mainapp.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+                />
+                {this.state.navlist[0].active ? <Home /> : ''}
+                {this.state.navlist[1].active ? <Charts /> : ''}
+                {this.state.navlist[2].active ? <Configure /> : ''}
+            </div>
+        );
+    }
+}
+
+export default MainApp
```
Comment (roshnee7) [inline src/Components/mainapp.js]: fix end of file

**Review by roshnee7 — CHANGES_REQUESTED** [review]
minor changes.

**Comment by nsk515** [comment]
@roshnee7 resolved. Please re-review.

File: src/Components/me_navbar.css [hunk 5]
```diff
@@ -22,32 +14,37 @@ ul {
...
+    padding-top: 8px;
+    box-shadow: rgba(0, 0, 0, 0.5) inset 2px 2px 2px;
   }
 
   .container {
     height : 100%;
-  }
\ No newline at end of file
+  }
+
```
Comment (roshnee7) [inline src/Components/me_navbar.css]: add new line at end of line

**Review by roshnee7 — APPROVED** [review]

**Pull request merged**